apps/blockchain/lib/blockchain/chain.ex

```diff
@@ -207,6 +222,31 @@ defmodule Blockchain.Chain do
     {address, account}
   end
 
+  defp get_builtin_account({raw_address, info}) do
+    address = load_raw_hex(raw_address)
+
+    balance = if info["balance"], do: load_decimal(info["balance"]), else: 0
+
+    nonce =
+      if info["nonce"],
+        do: load_hex(info["nonce"]),
+        else: 0
+
+    builtin_account = %{
```

Comment:
Would it be an improvement to turn this `builtin_account` into a struct and have a function that generates it from json (essentially this function you have here)?  What do you think?